packager/signer/signer.go

```diff
@@ -349,6 +350,7 @@ func (this *Signer) serveSignedExchange(resp http.ResponseWriter, fetchResp *htt
 	// should fetch an update (half-way between signature date & expires).
 	resp.Header().Set("Content-Type", "application/signed-exchange;v=b2")
```

Comment:
I wonder if these magic values should be replaced by const? E.g. when applicable, easier to change v=b2 to v=b3 at the top of the file instead of hunting around.

Also, maybe easier to understand if resp and fetchResp are renamed? outer? inner? sxgResp?